Sources/FlexLayout.swift

```diff
@@ -686,7 +686,7 @@ public final class Flex {
      This method is valid only when the item position is absolute (`view.flex.position(.absolute)`)
      */
     @discardableResult
-    public func vertically(_ percent: FPercent) -> Flex {
+    public func vertically(_ percent: FPercent = 0%) -> Flex {
```

Comment:
This method change is not required, the one with the `CGFloat` parameter will be enough to support this feature.